app/assets/stylesheets/application.scss

```diff
@@ -46,3 +46,23 @@ p {
   height: 1px;
   overflow: hidden;
 }
+
+.hide-bullet-points {
+  list-style: none;
+}
+
+.hide-bullet-points li{
+  margin: 5px 0;
+}
+
+.mobile-display-only{
+  color: white !important;
+  font-weight: bold;
+  margin-top: 20px;
+}
+
+@media (min-width: 48.0625em){ 
```

Comment:
I'm not a fan of the random values.  Did you get a chance to look at `govuk-media-query`?

Reply:
Resolved in 91ff03d